influxql/ast.go

```diff
@@ -1066,6 +1066,12 @@ type SelectStatement struct {
 	// Removes the "time" column from the output.
 	OmitTime bool
 
+	// Removes measurement name from resulting query. Useful for meta queries.
+	StripName bool
```

Comment:
I'm not a big fan of adding these two options, but I don't see an alternative at the moment.